interface/weno/markTx.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * weno rx mark tx.
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+$date = date("Y-m-d");
+$script = filter_input(INPUT_GET, "rx");
+
+$e_script = explode("-", $script);
+
+if ($e_script[0] === "NewRx") {
+    //See if the value is set
+    $check = "SELECT ntx FROM prescriptions WHERE id = ?";
+    $getVal = sqlStatement($check, array($e_script[1]));
+    $val = sqlFetchArray($getVal);
```

Comment:
Since you are just grabbing one item, wouldn't sqlQuery be better? (then don't need the sqlFetchArray step)

Reply:
Ok will do.